src/styles/blocks/card.scss

```
@@ -0,0 +1,77 @@
+.card{
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid ...;
```